CHANGELOG.md

```diff
@@ -142,6 +142,7 @@ We could not reconstruct _all_ changes, but we tried our best to make the most o
 - [Guestlist] Restrict user information shown on Google Maps according to their settings and never show the street details ... unless you are an administrator
 - [Board] Deletion of board failed without error (#861)
 - [Forms] Hours and Minutes are not getting pre-selected in date time fields, if they are single digit (#971)
+- [Msgsys] Fixed various warnings (#1103)
```

Comment:
```suggestion
```

Most of the warnings are introduced due to the PHP 8 upgrade. Hence, we did not add this to the changelog at all. Lets continue with this tradition :)